src/App.js

```diff
@@ -1,34 +0,0 @@
-import { Fragment, useState } from "react";
-
-import Title from "./components/Layout/Title";
-import Words from "./components/Words/Words";
-import Bravo from "./components/Bravo/Bravo";
-
-function App() {
-  const [modalIsShown, setModalIsShown] = useState(false);
-  const [clearedCards, setClearedCards] = useState({});
-
-  const closeModalHandler = () => {
-    setModalIsShown(false);
-    setClearedCards({});
-    window.location.reload();
-  };
-
-  const showModalHandler = () => {
-    setModalIsShown(true);
```

Comment:
This could also be done in-line, but abstracting it here like you did is nice if you want to add more logic to this action in the future.